packages/reporter/src/sessions/sessions.tsx

```diff
@@ -31,13 +31,11 @@ const SessionRow = ({ name, isGlobalSession, id, state, status, testId }: Sessio
           {isGlobalSession && <GlobeIcon className='global-session-icon' />}
           {name}
         </span>
-        <span className='session-tag'>
-          <Tag
-            customClassName='session-status'
-            content={status}
-            type={`${state === 'failed' ? 'failed' : 'successful'}-status`}
-          />
-        </span>
+        <Tag
```

Comment:
| Before | After |
|--|--|
|<img width="640" alt="Screen Shot 2022-11-02 at 4 24 09 PM" src="https://user-images.githubusercontent.com/14099737/199605489-c5c154c8-a2b6-405e-8fe8-c2d80e3da84a.png">| <img width="671" alt="Screen Shot 2022-11-02 at 4 24 43 PM" src="https://user-images.githubusercontent.com/14099737/199605490-1ff67a95-0e33-45ee-8a59-716811d92617.png"> |

Reply:
that before is very confusing 'failed' messaging